CefSharp/CefSharpSettings.cs

```diff
@@ -41,5 +41,10 @@ static CefSharpSettings()
         /// the event handlers are hooked in the static constructor for the ChromiumWebBrowser class
         /// </summary>
         public static bool ShutdownOnExit { get; set; }
+
+        /// <summary>
+        /// The proxy options that will be used for all connections
```

Comment:
A think we should expand on this, explain what's going on. Command line arguments are set, if you set username/password then `GetAuthCredentials` will be called automatically. Also note that if you use this option it's impossible to change the proxy at dynamically at runtime.